File: src/exercises/pov/typescript/template.ts
export class Tree {
  constructor(public value: string, public children: Tree[] = []) {}

  public fromPov(target: string): Tree {
    // Your code here
    return this;
  }

  public pathTo(from: string, to: string): string[] {
    // Your code here
    return [];
  }
}

File: src/exercises/pov/go/template.go
package main

type Tree struct {
	Value    string
	Children []*Tree
}

func NewTree(value string, children ...*Tree) *Tree {
	return &Tree{Value: value, Children: children}
}

func FromPov(tr *Tree, target string) *Tree {
	// Your code here
	return tr
}

func PathTo(tr *Tree, from, to string) []string {
	// Your code here
	return nil
}
